src/main/java/com/snack/news/controller/AdminController.java

```diff
@@ -7,9 +7,11 @@
 import lombok.AllArgsConstructor;
 import org.springframework.data.domain.Page;
 import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused import - org.springframework.transaction.annotation.Transactional.](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4554642)